sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/StringUtilsSuite.scala

```diff
@@ -56,4 +56,32 @@ class StringUtilsSuite extends SparkFunSuite {
     assert(concat("1", "2") == "12")
     assert(concat("abc", "\n", "123") == "abc\n123")
   }
+
+  test("string concatenation with limit") {
+    def concat(seq: String*): String = {
+      seq.foldLeft(new StringConcat(7))((acc, s) => {acc.append(s); acc}).toString
```

Comment:
```
seq.foldLeft(new StringConcat(7)) { (acc, s) => acc.append(s); acc }.toString
```

Reply:
Will also fix line L49 above to be consistent.